src/main/java/Assignment7.java

```diff
@@ -0,0 +1,20 @@
+import java.io.BufferedReader;
+import java.io.IOException;
+import java.io.InputStreamReader;
+
+public class Assignment7 {
+    public static void main(String[] args) throws IOException {
```

Comment:
Instead of writing this code in void main, lets define a function for this 
 and lets create test in src/test/* package which tests your function